### PR TITLE
Wire up tx events

### LINKED ISSUE #2700: UI needs to wait until SDK is ready
This is an issue where the UI is making a request before SDK has processed the data. UI needs to wait for SDK to get sync'd

### PR DESCRIPTION
**Pull request by bthaile**
This PR relies on this PR https://github.com/AugurProject/augur/pull/2716

fixes https://github.com/augurproject/augur/issues/2700

Wired up transaction lifecycle hooks for user created open orders and cancel orders. Using existing state redux stores.

### CODE REVIEW

**Comment by bthaile** [comment]
still working, closing for now

**Review by brianosaurus — COMMENTED** [review]
I think this needs to be updated with my latest PR/master. Otherwise it looks good.

**Comment by bthaile** [comment]
Can do. I’ll update

updated: I merged in master didn't see any additional changes.

File: packages/augur-ui/src/modules/events/actions/transaction-conversions.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { UIOrder } from 'modules/types';
+import { convertOnChainPriceToDisplayPrice, convertOnChainAmountToDisplayAmount, numTicksToTickSize, QUINTILLION } from '@augurproject/sdk/build';
+import { SELL, BUY, TX_TRADE_GROUP_ID, TX_MARKET_ID, TX_OUTCOME_ID, TX_PRICE, TX_AMOUNT, TX_DIRECTION } from 'modules/common/constants';
+import { MarketInfo } from '@augurproject/sdk/build/state/getter/Markets';
+import { createBigNumber } from 'utils/create-big-number';
+
+export function convertTransactionOrderToUIOrder(onChainOrder, status: string, market: MarketInfo): UIOrder {
+  console.log(JSON.stringify(onChainOrder));
```
Comment (andrewdamelio) [inline packages/augur-ui/src/modules/events/actions/transaction-conversions.ts:8]: do we wanna keep this log?

**Review by andrewdamelio — APPROVED** [review]
Cancel events works really well!

**Pull request merged**